packages/frontend/src/features/tableCalculation/components/SqlForm.tsx

```diff
@@ -41,8 +48,42 @@ const SqlEditor = styled(AceEditor)<
               `}
 `;
 
+const SqlEditorActions: FC<{
+    isSoftWrapEnabled: boolean;
+    onToggleSoftWrap: () => void;
+}> = ({ isSoftWrapEnabled, onToggleSoftWrap }) => (
+    <div
+        style={{
+            position: 'absolute',
+            bottom: 0,
+            right: 0,
+            // Avoids potential collision with ScrollArea scrollbar:
+            marginRight: '5px',
+        }}
```

Comment:
another non-blocking, but you can use Mantine's `Box` and pass these styles as props